### PR TITLE
Regwall: i18n

### PR DESCRIPTION
**Pull request by ChrisAntaki**

### CODE REVIEW

File: assets/i18n/strings/pt_BR.xlb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<localizationbundle locale="pt-BR">
+<messages>
+
+
+
+
+<msg desc="This title text introduces people to the idea that Google helps them get more (free articles)." hidden="0" name="GAA_REGWALL_TITLE" xml:space="default">Veja mais com o Google</msg>
+<msg desc="This description text tells people they are out of free articles from a given publication. Then it tells people they can unlock more free articles if they create a publisher account with their Google account." hidden="0" name="GAA_REGWALL_DESCRIPTION" xml:space="default"><strong></strong>This article usually requires payment, but you can keep reading, compliments of Google, when you register with <ph name="PUBLICATION"><ex>AP News</ex>{publication}</ph> using your Google account.</msg>
```
Comment (elijahsoria) [inline assets/i18n/strings/pt_BR.xlb:9]: I see that this msg is in English whereas the other two are in Portuguese. Is that expected? The same thing is true with fr.xlb.

File: assets/i18n/strings/compile.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+async function main() {
+  const files = await fs.readdir(__dirname);
+  const xlbFiles = files.filter((name) => /\.xlb$/.test(name));
+  const localesPerMessage = {};
+  for (const xlbFile of xlbFiles) {
+    const xml = await fs.readFile(__dirname + '/' + xlbFile, 'utf8');
+    const $ = cheerio.load(xml);
+    let locale = $('localizationbundle').attr('locale').toLocaleLowerCase();
+    if (locale === 'en-gb') {
```
Comment (elijahsoria) [inline assets/i18n/strings/compile.js:30]: What about if locale.startsWith('en-')? Will we have to do the same thing for 'en-us'?

Comment (ChrisAntaki) [reply assets/i18n/strings/compile.js:30]: We're only expecting this one English variant

File: assets/i18n/strings/pt_BR.xlb (hunk 1 above)
Comment (ChrisAntaki) [reply assets/i18n/strings/pt_BR.xlb:9]: Yep! This message is newer and hasn't been translated yet

File: assets/i18n/strings/compile.js (hunk 2 above)
Comment (ChrisAntaki) [reply assets/i18n/strings/compile.js]: Checking for all "en-" locales wouldn't hurt though. I just updated the code to do that :)

**Review by elijahsoria — APPROVED** [review]

**Pull request merged**